testdata/cheats/Vm.sol

```diff
@@ -245,7 +250,7 @@ interface Vm {
     function record() external;
 
     // Gets all accessed reads and write slot from a recording session, for a given address
-    function accesses(address) external returns (bytes32[] memory reads, bytes32[] memory writes);
+    function accesses(address) external returns (bytes32[] memory, bytes32[] memory);
```

Comment:
cc @DaniPopes unsure why, but, encoding `accesses` using this tuple fails. If the return type is switched to `RecordedAccesses` (which should be the same), this cheatcode works. Might be missing something obvious as it's late here, but do you have any idea why?

Reply:
Do you have a test case for this, like raw data

Reply:
friendly ping @DaniPopes

Reply:
`encode` is wrong, it should be `encode_params` in `fn accesses`